src/EditorFeatures/Core.Wpf/Interactive/InteractiveEvaluator.cs

```diff
@@ -34,24 +26,21 @@
 namespace Microsoft.CodeAnalysis.Editor.Interactive
 {
     using InteractiveHost::Microsoft.CodeAnalysis.Interactive;
-    using RelativePathResolver = Scripting::Microsoft.CodeAnalysis.RelativePathResolver;
 
-    internal abstract class InteractiveEvaluator : IResettableInteractiveEvaluator
+    // TODO: Rename to InteractiveEvaluator https://github.com/dotnet/roslyn/issues/6441
+    // The code is not specific to C#, but Interactive Window has hardcoded "CSharpInteractiveEvaluator" name.
```

Comment:
>That'll keep the rest of the code sane to look at.
 
 It would not. We would still have a couple of places that need to use the concrete type (e.g. constructor). Since there are only 4 use cases total I don't think it's worth half hiding it.